gm4_tunnel_bores/data/gm4_tunnel_bores/functions/bore/direction/update_rotation.mcfunction

```diff
@@ -2,6 +2,11 @@
 # at world spawn
 # called by gm4_tunnel_bores:update_direction
 
+execute if entity @s[tag=!gm4_tunnel_bore_south] if score @s gm4_bore_data matches 0 at @s run function gm4_tunnel_bores:bore/direction/update_rotation_south
+execute if entity @s[tag=!gm4_tunnel_bore_west] if score @s gm4_bore_data matches 90 at @s run function gm4_tunnel_bores:bore/direction/update_rotation_west
+execute if entity @s[tag=!gm4_tunnel_bore_east] if score @s gm4_bore_data matches -90 at @s run function gm4_tunnel_bores:bore/direction/update_rotation_east
+execute if entity @s[tag=!gm4_tunnel_bore_north] if score @s gm4_bore_data matches 180 at @s run function gm4_tunnel_bores:bore/direction/update_rotation_north
+
 execute if score @s gm4_bore_data matches 0 at @s run tp @s ~ ~ ~ 0 ~
```

Comment:
I believe these tp commands can go in `gm4_tunnel_bores:bore/direction/update_rotation_<N/E/S/W>`

Reply:
I tried that, but this caused a bug when detecting where to place rails for some reason. I'll show you a video of what happened earlier. I'll try again and investigate.

Reply:
Yeah, I tried again and it's still there :/. I really have no idea why that change would have that effect. I only tested when changing directions to south, but it was enough to make me do things this way since i had no idea what the origin of the issue was (btw, took me forever to figure out exactly what change caused it since I couldn't see the correlation between the change and the effect ;__;)